Unit tests are failing after immutable core id PR has landed

The following tests need to be adapted and fixed to be compatible with changes that landed on this PR https://github.com/kytos-ng/kytos/pull/243, it's a matter of adapting the mocked objects:

=========================== short test summary info ============================
FAILED tests/unit/test_main.py::TestMain::test_create_schedule - AssertionErr...
FAILED tests/unit/test_main.py::TestMain::test_evc_from_dict_links - Attribut...
FAILED tests/unit/test_main.py::TestMain::test_evc_from_dict_paths - Attribut...
FAILED tests/unit/test_main.py::TestMain::test_update_circuit - AssertionErro...
FAILED tests/unit/models/test_path.py::TestPath::test_is_valid - AttributeErr...

--- a/tests/unit/models/test_path.py
+++ b/tests/unit/models/test_path.py
@@ -4,6 +4,7 @@ from unittest import TestCase
 from unittest.mock import patch, Mock
 
 from kytos.core.common import EntityStatus
+from kytos.core.switch import Switch
 
 # pylint: disable=wrong-import-position
 
@@ -197,12 +198,12 @@ class TestPath(TestCase):
 
     def test_is_valid(self):
         """Test is_valid method."""
-        switch1 = "00:00:00:00:00:00:00:01"
-        switch2 = "00:00:00:00:00:00:00:02"
-        switch3 = "00:00:00:00:00:00:00:03"
-        switch4 = "00:00:00:00:00:00:00:04"
-        switch5 = "00:00:00:00:00:00:00:05"
-        switch6 = "00:00:00:00:00:00:00:06"
+        switch1 = Switch("00:00:00:00:00:00:00:01")
+        switch2 = Switch("00:00:00:00:00:00:00:02")
+        switch3 = Switch("00:00:00:00:00:00:00:03")
+        switch4 = Switch("00:00:00:00:00:00:00:04")
+        switch5 = Switch("00:00:00:00:00:00:00:05")
+        switch6 = Switch("00:00:00:00:00:00:00:06")
 
         links1 = [
             get_link_mocked(switch_a=switch1, switch_b=switch2),

--- a/tests/unit/test_main.py
+++ b/tests/unit/test_main.py
@@ -220,7 +220,8 @@ class TestMain(TestCase):
 
         Verify object creation with circuit data and schedule data.
         """
-        _get_interface_by_id_mock.return_value = True
+
+        _get_interface_by_id_mock.return_value = get_uni_mocked().interface
         _validate_mock.return_value = True
         uni_from_dict_mock.side_effect = ["uni_a", "uni_z"]
         payload = {
@@ -269,7 +270,7 @@ class TestMain(TestCase):
 
         Verify object creation with circuit data and schedule data.
         """
-        _get_interface_by_id_mock.return_value = True
+        _get_interface_by_id_mock.return_value = get_uni_mocked().interface
         _validate_mock.return_value = True
         uni_from_dict_mock.side_effect = ["uni_a", "uni_z"]
         payload = {
@@ -866,7 +867,7 @@ class TestMain(TestCase):
     def _uni_from_dict_side_effect(self, uni_dict):
         interface_id = uni_dict.get("interface_id")
         tag_dict = uni_dict.get("tag")
-        interface = Interface(interface_id, "0", "switch")
+        interface = Interface(interface_id, "0", MagicMock(id="1"))
         return UNI(interface, tag_dict)
 
     @patch("apscheduler.schedulers.background.BackgroundScheduler.add_job")
@@ -1257,12 +1258,15 @@ class TestMain(TestCase):
             evc_as_dict_mock,
             uni_from_dict_mock,
             evc_deploy,
-            *mocks,
+            _,
+            interface_by_id_mock,
+            _,
+            _,
+            _,
             requests_mock,
         ) = args
 
-        for mock in mocks:
-            mock.return_value = True
+        interface_by_id_mock.return_value = get_uni_mocked().interface
         unis = [
             get_uni_mocked(switch_dpid="00:00:00:00:00:00:00:01"),
             get_uni_mocked(switch_dpid="00:00:00:00:00:00:00:02"),

